gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/write.ts

```diff
@@ -124,6 +125,59 @@ export default class WriteOffchainConfig extends SolanaCommand {
     return makeSharedSecretEncryptions(gauntletSecret!, operatorsPublicKeys)
   }
 
+  validateInput = (input: Input): boolean => {
```

Comment:
Consider extracting this fn as it's network agnostic and could be useful for other OCR2 integrations.

Reply:
Noted